keyboards/doio/kb38/keymaps/default/keymap.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2021 Katrina (@PepperKats)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LAYERTWO 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* ┌───┐   ┌───────┐┌───┬───┬───┐┌───┬───┬───┐
+     * │ESC│   │KC_BSPC││F1 │F2 │F3 ││F4 │F5 │F6 │
+     * └───┘   └───────┘└───┴───┴───┘└───┴───┴───┘
+     * ┌───┬───┬───┬───┐┌───┬───┬───┐
+     * │NUM│ / │ * │ - ││PAS│SCR│PSC│
+     * ├───┼───┼───┼───┤├───┼───┼───│
+     * │ 7 │ 8 │ 9 │   ││INS│HOM│PGU│
+     * ├───┼───┼───│ + │├───┼───┼───│
+     * │ 4 │ 5 │ 6 │   ││END│DEL│PGD│
+     * ├───┼───┼───┼───┤└───┼───┼───┘
+     * │ 1 │ 2 │ 3 │ E │    │UP │    
+     * ├───┼───┼───│ N │┌───┼───┼───┐┌───┬───┬───┐
+     * │   0   │DEL│ T ││LFT│DWN│RHT││ O │ O │ O │
+     * └───┴───┴───┴───┘└───┴───┴───┘└───┴───┴───┘
+     */
+    [_QWERTY] = LAYOUT(
+        MO(1),            KC_BSPC,          RGB_RMOD, RGB_TOG, RGB_MOD,   KC_F1,     KC_F2,     KC_F3,
+        KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS, KC_PAUS,  KC_SCRL, KC_PSCR, 
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS, KC_INS,   KC_HOME, KC_PGUP, 
+        KC_P4,   KC_P5,   KC_P6,            KC_END,   KC_DEL,  KC_PGDN, 
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,           KC_UP,
+        KC_P0,            KC_PDOT,          KC_LEFT,  KC_DOWN, KC_RGHT,  KC_HOME,    KC_END,    KC_B
+    ),
+    [_LAYERTWO] = LAYOUT(
+        KC_ESC,           KC_BSPC,          RGB_RMOD, RGB_TOG, RGB_MOD,   KC_A,   QK_REBOOT,   QK_BOOTLOADER,
```

Comment:
Momentary layer change keys need a transparent key below them.
```suggestion
        _______,           KC_BSPC,          RGB_RMOD, RGB_TOG, RGB_MOD,   KC_A,   QK_REBOOT,   QK_BOOTLOADER,
```